app/src/main/java/com/pluu/webtoon/utils/Corutine.kt

```diff
@@ -0,0 +1,20 @@
+package com.pluu.webtoon.utils
+
+import kotlinx.coroutines.experimental.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/63a0f11a30849185d2c70a8ffbd4f8b86fa84721/app/src/main/java/com/pluu/webtoon/utils/Corutine.kt#L3"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>